{{cookiecutter.project_name}}/.pre-commit-config.yaml

```
@@ -1,23 +1,82 @@
+fail_fast: false
+default_language_version:
+  python: python3
+default_stages:
+  - commit
+  - push
+minimum_pre_commit_version: 2.9.3
```

Update this?

@michalk8 says it's at 2.16 at least

```suggestion
minimum_pre_commit_version: 2.16.0
```